osquery/tables/system/darwin/crash_logs.cpp

```diff
@@ -0,0 +1,144 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include "osquery/tables/system/system_utils.h"
```

Comment:
Contrary to internal nits, we throw these at the bottom, the order is:
1. C
2. C++
3. System Libs
4. Library includes
5. Osquery
6. Local osquery